task-runner.sh

```diff
@@ -28,19 +28,20 @@ then
      if [ $TARGET_BRANCH == "master" ]
      then
      TEST_BRANCH="staging"
+     GULP_COMMANDS="gulp build_development"
      fi
      if [ $TARGET_BRANCH == "production" ]
      then
      TEST_BRANCH="master"
+     GULP_COMMANDS="gulp build"
      fi
  fi
 
 fi
 
-GULP_COMMANDS=""
```

Comment:
do the 2 above conditions capture all possible outcomes? Otherwise GULP_COMMANDS might be undefined